client/src/utils/helpers/notificationModalHelper.ts

```diff
@@ -0,0 +1,30 @@
+import { toTitleCase } from '~/utils/helpers/stringHelpers'
+import { INotification } from '~/utils/interfaces'
+import { STATUS_OPTIONS } from '~/utils/constants/notificationFilter'
+
+export const getNotificationTitle = (notification: INotification): string => {
+  const type = notification.type?.toLowerCase()
+  const name = notification.name
+  const status = notification.status
```

Comment:
```suggestion
  const { type = '', name, status, specificType } = notification
  const lowerType = type.toLowerCase()
  const typeParts = lowerType.split('_')
```

- Smaller footprint: fewer variable declarations.
- Less redundancy: split('_') only once.